content/en/docs/reference/config-api/apiserver-webhookadmission.v1.md

```diff
@@ -0,0 +1,46 @@
+---
+title: WebhookAdmission Configuration (v1)
+content_type: tool-reference
+package: apiserver.config.k8s.io/v1
+auto_generated: true
+---
+Package v1 is the v1 version of the API.
+
+## Resource Types 
+
+
+- [WebhookAdmission](#apiserver-config-k8s-io-v1-WebhookAdmission)
+  
+    
+
+
+## `WebhookAdmission`     {#apiserver-config-k8s-io-v1-WebhookAdmission}
+    
+
+
+
+
+WebhookAdmission provides configuration for the webhook admission controller.
```

Comment:
nice-to-have: make “webhook admission controller” a hyperlink. Also I think that that's actually 2 controllers (validating and mutating).

Reply:
This "nice-to-have" requires upstream code change. :)

Reply:
> upstream code change

We can do a different but similarly helpful thing purely in Hugo (and that's the approach I think I'd take). Will try to sketch something out _eventually_. No promises.